Classes/Controller/WorkspaceController.php

```diff
@@ -234,6 +234,14 @@ protected function listWorkspaceAction($checkedDocumentIdentifiers = [])
 
             $this->session->setWorkspaceData($workspaceSessionData);
         }
+        if ($args['errorMessage']) {
+            $this->addFlashMessage(
+                $args['errorMessage'],
+                //LocalizationUtility::translate("manager.workspace.bookmarkRemoved.singular", "dpf"),
```

Comment:
Message is commented out in this call. Either there is something missing, or the comment should be removed.

Reply:
I removed the comment.